Makefile

```diff
@@ -245,6 +245,14 @@ website-yaml:
 			-e "s|{{AMBASSADOR_DOCKER_IMAGE}}|$(AMBASSADOR_DOCKER_REPO):$(VERSION)|g;s|{{STATSD_DOCKER_IMAGE}}|$(STATSD_DOCKER_REPO):$(VERSION)|g" \
 			{} \;
 
+generate-rbac-yaml:
```

Comment:
This is a problem. `create-manifests.sh` should already be doing this, and `make e2e-versioned-manifests` calls `create-manifests.sh`.